framework/packages/abstract-testing/src/mock_querier.rs

```diff
@@ -51,11 +47,19 @@ pub struct MockQuerierBuilder {
     raw_handlers: HashMap<Addr, Box<RawHandler>>,
     raw_mappings: HashMap<Addr, HashMap<Binary, Binary>>,
     contract_admin: HashMap<Addr, Addr>,
+    // Used for Address generation
+    pub api: MockApi,
```

Comment:
Seems like the only reason this stores `MockApi` is for the `AbstractAddrs` getter in the trait.

Reply:
Yep, otherwise we can't get native addresses of abstract or pass MockApi on every method instead of only construction(`Default` is still available)